docs/two-phase-commit-transactions.md

```diff
@@ -183,9 +183,49 @@ Similar to `prepare()`, you can call `validate()` in the coordinator/participant
 Currently, you need to call `validate()` when you use the `Consensus Commit` transaction manager with `EXTRA_READ` serializable strategy in `SERIALIZABLE` isolation level.
 In other cases, `validate()` does nothing.
 
-### Suspend and resume the transaction
+### Load balancing in Two-phase Commit Transactions
+
+Usually, services using Two-phase Commit Transactions achieve a transaction while exchanging multiple requests/responses as follows:
+
+<p align="center">
+<img src="images/two_phase_commit_sequence_diagram.png" width="400" />
+</p>
+
+Also, a service typically has multiple hosts/servers for scalability and availability, and uses server-side (proxy) or client-side load balancing to distribute the requests to the hosts/servers.
+Since a transaction processing in Two-phase Commit Transactions is stateful, operations in a transaction must be executed in the same hosts/servers.
```

Comment:
```suggestion
In such a case, since a transaction processing in Two-phase Commit Transactions is stateful, requests in a transaction must be routed to the same servers while different transactions need to be distributed to balance the load.
```